packages/core/smoke-test/smoke-test.js

```diff
@@ -88,23 +90,38 @@ const copyTestApps = workdir => {
   TEST_APPS.forEach(appName => {
     const srcAppDir = path.join(repoRoot, 'example-apps', appName);
     const destAppDir = path.join(workdir, appName);
-    fs.copySync(srcAppDir, destAppDir);
+    fs.copySync(srcAppDir, destAppDir, {
+      filter: (src, dest) => !src.includes('node_modules/')
```

Comment:
This is a small bug I found when we copy example apps to temporary directories. Example apps could have a `node_module/` locally. We should avoid copying that to the temporary directory.